### PR TITLE
Add decomposition for expand_copy

### PR DESCRIPTION
**Pull request by rec**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #130940

* Extracted from #129476

cc @XilunWu @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

**Review by peterbell10 — APPROVED** [review]

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -13164,6 +13164,17 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            skips=(
                DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),),
            ),
+    OpInfo('expand_copy',
+           # op=lambda self, shape: self.expand(shape),
```
Comment (peterbell10) [inline torch/testing/_internal/common_methods_invocations.py:13168]: Is this stale from a copy-paste?

**Comment by rec** [comment]
topic: not user facing

File: torch/testing/_internal/common_methods_invocations.py (hunk 1 above)
Comment (rec) [reply torch/testing/_internal/common_methods_invocations.py]: Fixed! Still waiting for very slow CI to commit this.

Hope you're having fun out there!!

**Pull request closed without merging**